README.md

```diff
@@ -62,7 +62,7 @@ wiki.
 
 - [sharkdp/fd](https://github.com/sharkdp/fd) (finder)
 - [nvim-treesitter/nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) (finder/preview)
-- [neovim LSP]( https://neovim.io/doc/user/lsp.html) (picker)
+- [neovim LSP](https://neovim.io/doc/user/lsp.html) (picker)
```

Comment:
please no unrelated changes.

Please open a new PR for the proposed readme changes

Reply:
I force-push removed these unrelated changes.